.github/workflows/auto-assign-owners.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     if: ${{ github.actor != 'dependabot[bot]' }}
     steps:
       - name: run
-        uses: kentaro-m/auto-assign-action@v1.2.3
```

Comment:
Should we replace with `kentaro-m/auto-assign-action@v1` so we use the latest all the time?